Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -147,15 +145,20 @@ def tags_arg_to_request_format(tags):
         } for tag in tags]
     except IndexError:
         return_error(bad_arg_msg)
+    return None
 
 
 ''' INTEGRATION COMMANDS '''
 
 
-def test_connection(client, params):
-    if not client.ms_client.managed_identities_client_id \
-        and (params.get('self_deployed', False) and not params.get('client_credentials')
-             and not (params.get('credentials_auth_code', {}).get('password') or params.get('auth_code'))):
+def test_connection(client: Client, params: dict[str, Any]) -> str:
+    if (
+        not client.ms_client.managed_identities_client_id
+        and params.get('self_deployed', False)
+        and not params.get('client_credentials')
+        and not params.get('credentials_auth_code', {}).get('password')
+        and not params.get('auth_code')
```

Comment:
It is best practice to keep a comment explaining what are you checking here or improve the error returning as it is only explaining one condition. If possible I would take it out to another boolean function.

Reply:
This code was edited by lint, I didn't write it... 🫣